consolidates git provider options (#6691)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/git/git-configuration-in-dbt-cloud.md

---
title: "Configure Git in dbt Cloud"
description: "Learn about the Git providers supported in dbt Cloud"
hide_table_of_contents: true
pagination_next: "docs/cloud/git/managed-repository"
---

> [Version control](/docs/collaborate/git/version-control-basics) is an essential part of the dbt workflow, enabling teams to collaborate effectively and maintain a history of changes to their dbt projects. 

Reviewer: Not a blocker here, but in the future,e we might want to open with a little blurb about what Git is

> [Version control](/docs/collaborate/git/version-control-basics) &mdash; a system allows you and your teammates to work safely and simultaneously on a single project &mdash; is an essential part of the dbt workflow. It enables teams to collaborate effectively and maintain a history of changes to their dbt projects. 

Author: ```suggestion
[Version control](/docs/collaborate/git/version-control-basics) &mdash; a system that allows you and your teammates to work safely and simultaneously on a single project &mdash; is an essential part of the dbt workflow. It enables teams to collaborate effectively and maintain a history of changes to their dbt projects. 
```

> [Version control](/docs/collaborate/git/version-control-basics) &mdash; a system allows you and your teammates to work collaboratively, safely, and simultaneously on a single project &mdash; is an essential part of the dbt workflow, enabling teams to collaborate effectively and maintain a history of changes to their dbt projects. 

Author: ```suggestion
[Version control](/docs/collaborate/git/version-control-basics) &mdash; a system allows you and your teammates to work safely and simultaneously on a single project &mdash; is an essential part of the dbt workflow. It enables teams to collaborate effectively and maintain a history of changes to their dbt projects. 
```

In dbt Cloud, you can configure Git integrations to manage your dbt project code with ease. dbt Cloud offers multiple ways to integrate with you Git provider, catering to diverse team needs and preferences. 

Whether you use a Git integration that natively connects with dbt Cloud or prefer to work with a managed or cloned repository, dbt Cloud supports flexible options to streamline your workflow.

<div className="grid--3-col">

<Card
    title="Managed repository"
    body="Learn how to quickly set up a project with a managed repository."
    link="/docs/cloud/git/managed-repository"
    icon="dbt-bit"/>

<Card
    title="Git clone"
    body="Learn how to connect to a git repository using a git URL and deploy keys."
    link="/docs/cloud/git/import-a-project-by-git-url"
    icon="dbt-bit"/>

<Card
    title="Connect to GitHub"
    body="Learn how to connect to GitHub using dbt Cloud's native integration."
    link="/docs/cloud/git/connect-github"
    icon="dbt-bit"/>

<Card
    title="Connect to GitLab"
    body="Learn how to connect to GitLab using dbt Cloud's native integration."
    link="/docs/cloud/git/connect-gitlab"
    icon="dbt-bit"/>

<Card

>     title="Connect to Azure DevOps "

Author: ```suggestion
    title="Connect to Azure DevOps"
```
